pages/ImprovingAnExploration.html

```diff
@@ -16,7 +16,7 @@ <h2>
     "Exploration Stats" tab in the exploration editor. If any reader has viewed
     your exploration, you should see a page like this one:
   </p>
-  <p><img src="static/images/analytics.png" width="100%"></p>
+  <p><img src="static/images/analytics.png" width="100%" alt="An Analytics page conprising of a horizontal 'completion-rate' meter, depicting 'completions' in green and 'non-completions' in red. Below which is a flow chart depicting common paths learners have followed, from start to end."></p>
```

Comment:
comprising of --> comprising

Below which is --> This is followed by

"Flowchart" is one word.